src/components/Graph/Graph.jsx

```diff
@@ -1,119 +1,123 @@
 import React, { Component } from 'react';
 import { Line } from 'react-chartjs-2';
-import PropTypes from 'prop-types';
-import './Graph.css';
+import "./Graph.css";
 
-import loading from '../../img/loading.svg';
+import loading from "../../img/loading.svg";
 
 const offWhite = 'rgb(247, 248, 249)';
 const gray = '#929292';
 
 class Graph extends Component {
-  data = {
-    labels: this.props.labels,
-    datasets: [
-      {
-        fill: false,
-        borderColor: offWhite,
-        borderCapStyle: 'butt',
-        borderJoinStyle: 'miter',
-        borderWidth: 1,
-        pointRadius: 3,
-        pointBorderColor: offWhite,
-        pointBackgroundColor: offWhite,
-        pointHoverRadius: 10,
-        pointHoverBackgroundColor: offWhite,
-        pointHoverBorderWidth: 0,
-        pointHitRadius: 30,
-        lineTension: 0.4,
-        data: this.props.data,
-      },
-    ],
-  };
+   constructor(props) {
+      super(props);
+
+      this.state = {
+         data: this.getData()
+      }
+   }
+
+   componentDidUpdate(prevProps) {
+      if (prevProps.labels !== this.props.labels) {
+         this.setState({data: this.getData()});
+      }
+   }
+
+   getData() {
```

Comment:
nit: getDataConfig

Reply:
👍

Reply:
This function is actually get new data from the props so calling it getDataConfig would be a little off. But I do think I need to extract the config stuff to a separate function.